applications/software-libraries.md

```diff
@@ -0,0 +1,59 @@
+# Software Libraries
+
+The following describes the approach we adopt when making use of external software libraries in our applications.
+
+## Centrally capture your application's dependencies *in code*
+
+For Ruby this will primarily be in your **Gemfile** or equivalent, for Node in your **package.json** file, for Java in your **POM** file.
+
+This may seem like stating the obvious, but it is essential to allow the application to be checked for any issues in libraries that it depends on.
+
+## Capture your client-side dependencies too
+
+Ensure that any client-side libraries you use are captured in your dependencies.
+
+Again, this sounds obvious but not all client-side frameworks and libraries are necessarily included in your application through bundler or npm.
+
+> As an example: The GOV.UK frontend toolkit includes the GPL-licensed Nomensa accessible media player.  If you link to this in your client-side Javascript you will have to GPL *all* your client-side code.  Does the toolkit's MIT licence cover this?  No.  Is this dependency listed anywhere in the toolkit?  No.  Mind you, IANAL so maybe none of this matters.
```

Comment:
I think we want the guides to be clear on key points. So if we are stating that 

> If you link to [Nomensa] in your client-side Javascript you will have to GPL *all* your client-side code

then I would suggest dropping at least the last line

> Mind you, [I am not a lawyer] so maybe none of this matters.

or perhaps even everything after that statement.